.github/workflows/pr-clean-deployments.yml

```diff
@@ -26,33 +26,20 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           registry-url: 'https://registry.npmjs.org'
-      - run: |
+      - name: Setup git user
+        run: |
           git config user.name github-actions
           git config user.email github-actions@github.com
+      - name: Remove pr storybook deployment
+        if: "contains(github.event.pull_request.labels.*.name, 'Build: React Contoso')"
```

Comment:
This will trigger only for react-contoso. I assume it should be Build: Storybook?

Reply:
oops, fixing that now.